sdk/textanalytics/Azure.AI.TextAnalytics/tests/MockTests/Legacy/AnalyzeOperationV30MockTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Legacy.Tests
+{
+    public class AnalyzeOperationV30MockTests : ClientTestBase
```

Comment:
If we did find some way to centralize min versions, it'd be great to data-drive these so that we always have coverage when making changes.

Reply:
Agreed, but seems beyond the scope of this PR, especially given timing. I'm pretty diligent in my testing, so lets hope at the very least future maintainers are as well.